dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/publish/servers/Telemetry10JaxWithConcurrency/bootstrap.properties

```diff
@@ -1 +1,5 @@
 bootstrap.include=../testports.properties
+# beta fence for HTTP tracing as this has tests that expect output from the servlet filter
+-Dcom.ibm.ws.beta.edition=true
```

Comment:
This option isn't actually doing anything. The line would be correct if it was in jvm.options, but it won't work in bootstrap.properties like this.

If we're taking out all the beta repeats anyway, we can just remove this line.